complete/contract-rest-service/src/test/java/hello/BaseClass.java

```diff
@@ -0,0 +1,28 @@
+package hello;
+
+import org.junit.Before;
+import org.junit.runner.RunWith;
+import org.mockito.Mockito;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.test.context.junit4.SpringRunner;
+
+import io.restassured.module.mockmvc.RestAssuredMockMvc;
+
+@RunWith(SpringRunner.class)
+@SpringBootTest(classes = ContractRestServiceApplication.class)
+public abstract class BaseClass {
```

Comment:
I think we can simplify this to the following (we don't need to start the whole context to run the contract tests)

```java
@RunWith(MockitoJUnitRunner.class)
public abstract class BaseClass {

	@InjectMocks PersonRestController personRestController;
        @Mock PersonService personService;

	@Before public void setup() {
		RestAssuredMockMvc.standaloneSetup(personRestController);

		Mockito.when(personService.findPersonById(1L))
				.thenReturn(new Person(1L, "foo", "bee"));
	}

}
```

Reply:
thanks @marcingrzejszczak. It works perfectly, I am adapting this change to the officially released guide.